### ISSUE TITLE
SAM not listing any of my games and giving me an error

### ISSUE THREAD

**Issue by SENRelax** [gibbed/SteamAchievementManager#434]
Hi, so I used SAM yesterday and it was working fine and I went to use it today and nothing I do makes it work. SAM just gives me a black screen with none of my games on there and open an error message everytime. Please Help.
![SAM](https://github.com/user-attachments/assets/b09a55a1-775f-46ba-8925-368842055c2f)

**Comment by SENRelax** [comment]
> I'm having this issue too. Guessing Steam have changed something in the background?

Yeah its really annoying because, there are achievements I really need this for!

**Comment by Apolosclei** [comment]
Same issue here

**Comment by Damaux** [comment]
Same here^

**Comment by Kasyrgan1** [comment]
cant load any games aswell after joining my brother in a family. is that maybe the cause?

**Comment by LiamW10** [comment]
You can still find the game you want to add trophies to. Just type the AppID for the game you want to change and then click the "Add Game" button. It will then appear and should have all of the achievements that you already own ticked. If you don't know the AppID, you can find it by going into you Steam library, finding the game, going into the properties and then into the update section. You will then find the App ID at the bottom of the window.
